keyboards/free_willy/rules.mk

```diff
@@ -0,0 +1 @@
+
```

Comment:
This needs MCU and basic configuration settings

Reply:
done, thank you!